packages/react-error-overlay/src/index.js

```diff
@@ -8,15 +8,15 @@
  */
 
 /* @flow */
-import React from 'react';
-import type { Element } from 'react';
-import ReactDOM from 'react-dom';
-import CompileErrorContainer from './containers/CompileErrorContainer';
-import RuntimeErrorContainer from './containers/RuntimeErrorContainer';
 import { listenToRuntimeErrors } from './listenToRuntimeErrors';
-import { iframeStyle, overlayStyle } from './styles';
+import { iframeStyle } from './styles';
 import { applyStyles } from './utils/dom/css';
 
+/* eslint-disable import/no-webpack-loader-syntax */
+//$FlowFixMe
+import iframeScript from 'raw-loader!./bundle.js';
```

Comment:
This hack is fine for now IMO, as long as we switch it eventually (once we iron out how to handle raw files).